cattle/plugins/docker/storage.py

```diff
@@ -112,7 +111,4 @@ def _do_volume_remove(self, volume, storage_pool, progress):
                     shutil.rmtree(path)
 
     def _path_to_volume(self, volume):
-        host_path = volume.uri.replace('file://', '')
-        mounted_path = re.sub('^.*?/var/lib/docker', '/host/var/lib/docker',
-                              host_path)
-        return mounted_path
+        return volume.uri.replace('file://', '')
```

Comment:
Yeah, my head went to the same place: orient from the vfs dir.

Do you think we should open a bug with boot2docker or docker for this? It feels like docker inspect **should** be reporting this as /var/lib/docker (or whatever the --graph option was set to when the daemon was started?)

To that point, some potentially useful info while thinking about this:
```
$ ps -ef | grep docker
 1197 root     /usr/local/bin/docker -d -D -g /var/lib/docker -H unix:// -H tcp://0.0.0.0:2375 --tlsverify --tlscacert=/var/lib/boot2docker/ca.pem --tlskey=/var/lib/boot2docker/server-key.pem --tlscert=/var/lib/boot2docker/server.pem -H tcp://0.0.0.0:2376
...
$ docker info
Containers: 3
Images: 37
Storage Driver: aufs
 Root Dir: /mnt/sda1/var/lib/docker/aufs
 Dirs: 57
Execution Driver: native-0.2
Kernel Version: 3.16.7-tinycore64
Operating System: Boot2Docker 1.4.1 (TCL 5.4); master : 86f7ec8 - Tue Dec 16 23:21:51 UTC 2014
CPUs: 8
Total Memory: 999.6 MiB
Name: craig1
ID: 736S:EUJK:KWU2:XSS6:UCCY:CO5Y:46GH:3VXM:EGP5:K7ZL:PCRZ:6PE4
Debug mode (server): true
Debug mode (client): false
Fds: 32
Goroutines: 29
EventsListeners: 0
Init Path: /usr/local/bin/docker
Docker Root Dir: /mnt/sda1/var/lib/docker
```